docs/assets/js/sidebar.js

```diff
@@ -5,15 +5,26 @@
 function init() {
   const secondaryNavCat = document.querySelector( '.ds-nav-container' );
 
+  // First collapse the navigation if in mobile.
+  if ( innerWidth < 601 ) {
```

Comment:
For clarity maybe make this `window.innerWidth` to emphasize you're checking the global value.

Reply:
Or actually, couldn't you move `let windowWidth = window.innerWidth;` above this line and use `windowWidth` here?